src/public/js/product.js

```diff
@@ -175,10 +195,66 @@ modalClose.addEventListener('click', function () {
   modal.close();
 });
 
-clonedForm.addEventListener('submit', function () {
-  modal.close();
-});
-
 function isMobile() {
   return window.innerWidth < 1024;
 }
+
+const errorMap = [
+  'El número de teléfono proporcionado no es válido',
+  'Código de país no válido',
+  'El número de teléfono es demasiado corto.',
+  'El número de teléfono es demasiado largo',
+  'El número de teléfono proporcionado no es válido',
+];
+
+const commonCustomViewErrors = {
+  add: function (field, message, cls) {
+    let error = document.querySelector(`.${field.id}.${cls}`) || document.createElement('p');
+    error.textContent = message;
+
+    if (!document.querySelector(`.${field.id}.${cls}`)) {
+      error.classList.add(field.id, cls);
+      clonedForm.querySelector(`[for='${field.id}']`).after(error);
+    }
+  },
+  remove: function (field, cls) {
+    clonedForm.querySelector(`.${field.id}.${cls}`)?.remove();
+  },
+};
+
+const MESSAGES = {
+  required: 'campo obligatorio',
+  email: 'correo inválido',
+};
+
+const SELECTORS = {
+  messageError: 'modal-form__error',
+};
+
+const configValidator = {
+  container: '.modal-form',
+  validateOnFieldChanges: true,
+  button: '.modal-form__submit',
+  customValidates: {
+    intlTelInput: {
+      message: errorMap[0],
+      // eslint-disable-next-line
+      fn: function (field, container) {
```

Comment:
Remove this line and remove unused variable
```suggestion
      fn: function (field) {
```